repos/c2sm/packages/py-scikit-build-core/package.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+from spack.pkg.builtin.py_scikit_build_core import PyScikitBuildCore as SpackPyScikitBuildCore
+
+
+class PyScikitBuildCore(SpackPyScikitBuildCore):
```

Comment:
same here

Reply:
I'm not sure if Magdalena required 0.5.0. The inbuilt has v0.2.0. Can try without it perhaps.

Reply:
Checked with @halungge, and it seems `ghex` definitely needs a more recent version of `scikit-build-core` to support certain cmake-related options in pyproject.toml.